example/script.js

```diff
@@ -57,4 +57,35 @@ window.onload = async () => {
         copyUInt[i] = i;
     }
     await worker.call("time-transfer", [new Date(), { complexObject: copyUInt }]);
+
+    // Try send function Error or DOM Element
+    try {
+        await worker.call("log", [function test() {}]);
+    } catch (e) {
+        console.log(e);
+    }
+
+    try {
+        await worker.call("log", [new Error("TEST")]);
+    } catch (e) {
+        console.log(e);
+    }
+
+    try {
+        await worker.call("log", [document.createElement("p")]);
+    } catch (e) {
+        console.log(e);
+    }
+
+    try {
+        await worker.call("log", [{
+            a: {
+                b: [
+                    0, 1, function test() {},
+                ],
+            },
+        }]);
+    } catch (e) {
+        console.log(e);
+    }
```

Comment:
all those try-catches could be as one

Reply:
ah. no.

Reply:
Hello @jbghoul :D,
I am not sure, we want to show an error by example (One for Error, One for Function and One for HTMLElement).

If we added all example in one try catch we will have just one log.

But this part of repo is just for the example, we can make like you want, this part will soon evolve for real tests !